### PR TITLE
Add test for empty strings

### PR DESCRIPTION
**Pull request by gerijeb**
Pass empty string as part of the arguments to test.

### CODE REVIEW

File: 05_sumAll/sumAll.spec.js [hunk 1]
```diff
@@ -19,4 +19,7 @@ describe('sumAll', () => {
   test.skip('returns ERROR with non-number parameters', () => {
     expect(sumAll(10, [90, 1])).toEqual('ERROR');
   });
+  test.skip('returns ERROR with empty strings', () => {
```
Comment (twalton83) [inline 05_sumAll/sumAll.spec.js:22]: 'returns ERROR with non-number parameters' covers this test case.
